src/metadata/trackmetadata.h

```diff
@@ -1,5 +1,5 @@
-#ifndef TRACKMETADATA_H
-#define TRACKMETADATA_H
+#ifndef MIXXX_TRACKMETADATA_H
```

Comment:
We don't normally prefix the guards with anything other than the filename. Did this conflict with something?